src/test/java/api/support/matchers/ScheduledNoticeMatchers.java

```diff
@@ -0,0 +1,66 @@
+package api.support.matchers;
+
+import static api.support.matchers.TextDateTimeMatcher.isEquivalentTo;
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.hasJsonPath;
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.withJsonPath;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.is;
+
+import java.util.UUID;
+
+import org.folio.circulation.domain.policy.Period;
+import org.hamcrest.Matcher;
+import org.hamcrest.Matchers;
+import org.joda.time.DateTime;
+
+import io.vertx.core.json.JsonObject;
+
+public class ScheduledNoticeMatchers {
+
+  private static final String LOAN_ID = "loanId";
+  private static final String NEXT_RUN_TIME = "nextRunTime";
+  private static final String TIMING = "noticeConfig.timing";
+  private static final String RECURRING_PERIOD_INTERVAL = "noticeConfig.recurringPeriod";
+  private static final String INTERVAL_ID = "intervalId";
+  private static final String DURATION = "duration";
+  private static final String TEMPLATE_ID = "noticeConfig.templateId";
+  private static final String FORMAT = "noticeConfig.format";
+  private static final String SEND_IN_REAL_TIME = "noticeConfig.sendInRealTime";
+
+  public static Matcher<JsonObject> hasScheduledLoanNoticeProperties(
```

Comment:
I think the `Properties` postfix can be removed from this method name. At this level, the matcher is checking the notice, rather than specifically the properties.

```suggestion
  public static Matcher<JsonObject> hasScheduledLoanNotice(
```

Reply:
Done